build failures seems related
```
[  8%] Building C object libs-native/System.Security.Cryptography.Native/CMakeFiles/objlib.dir/pal_ssl.c.o
    [  8%] Building C object libs-native/System.Security.Cryptography.Native/CMakeFiles/objlib.dir/pal_pkcs7.c.o
    /__w/1/s/artifacts/source-build/self/src/src/native/libs/System.Security.Cryptography.Native/pal_ssl.c:165:5: error: 'NEED_OPENSSL_3_0' is not defined, evaluates to 0 [-Werror,-Wundef]
    #if NEED_OPENSSL_3_0
        ^
    1 error generated.
```

--- a/src/native/libs/System.Security.Cryptography.Native/pal_ssl.c
+++ b/src/native/libs/System.Security.Cryptography.Native/pal_ssl.c
@@ -162,7 +162,7 @@ SSL_CTX* CryptoNative_SslCtxCreate(const SSL_METHOD* method)
         // to be to use server preference (as of June 2020), so just always assert that.
         SSL_CTX_set_options(ctx, SSL_OP_NO_COMPRESSION | SSL_OP_CIPHER_SERVER_PREFERENCE);
 
-#if NEED_OPENSSL_3_0
+#ifdef NEED_OPENSSL_3_0
         if (CryptoNative_OpenSslVersionNumber() >= OPENSSL_VERSION_3_0_RTM)
         {
             // OpenSSL 3.0 forbids client-initiated renegotiation by default. To avoid platform

